lib/commands/run-test-scenario.ts

```diff
@@ -88,13 +90,32 @@ export const builder: yargs.CommandBuilder = {
   cleanUp: {
     describe: "whether delete resource group when all steps finished",
     boolean: true,
-    default: true,
+    default: false,
   },
   dryRun: {
     describe: "dry run mode. only create postman collection file not run live api test.",
     boolean: true,
     default: false,
   },
+  from: {
+    describe:
+      "the step to start with in current run, it's used for debugging and make sure not use --cleanUp to delete resource group in the previous run.",
+    string: true,
+    demandOption: false,
+    implies: "runId",
+  },
+  to: {
+    describe:
+      "the step to end in current run,it's used for debugging and make sure not use --cleanUp to delete resource group in the previous run.",
+    string: true,
+    demandOption: false,
+    implies: "runId",
+  },
+  runId: {
+    describe: "specify the last runId for debugging",
```

Comment:
Why the last? We could support any runId if current local env storage exist.

Reply:
Good capture, I'll remove the 'last' from the description